perceval/backends/jira.py

```diff
@@ -163,6 +167,34 @@ def parse_issues(raw_page):
         for issue in issues:
             yield issue
 
+    @staticmethod
```

Comment:
In general, I don't like the idea of changing the status of an object inside of a function. It can drive to future problems. I think more in something like:

```python

def map_custom_field(custom_fields, fields):
    build_cf = lambda cf, v: {'id': cf['id'], 'name': cf['name'], 'value' : v}

    return  {k : build_cf(custom_fields[k], v) \
             for k, v in fields.items() \
             if k in custom_fields}

mapping = map_custom_field(custom_fields, issue['fields'])
for k, v in mapping.items():
    issue['fields'][k] = v
```

Regarding the other function, don't use the underscore variable, this usually means you aren't going to use it.